### ISSUE TITLE
Android 11 support?

### ISSUE THREAD

**Issue by craig-at-rsg** [ionic-team/ionic-storage#196]
While testing my app against the Android 11 Beta, I got the following console logging before my app white-screened:

```
plugins/cordova-sqlite-storage/www/SQLitePlugin.js:175 OPEN database: _ionicstorage
plugins/cordova-sqlite-storage/www/SQLitePlugin.js:106 new transaction is queued, waiting for open operation to finish
plugins/cordova-sqlite-storage/www/SQLitePlugin.js:197 OPEN database: _ionicstorage FAILED, aborting any pending transactions
plugins/cordova-sqlite-storage/www/SQLitePlugin.js:83 Could not open database
```

Has this plugin been tested against Android 11?

**Comment by mhartington** [comment]
this is not specific to this plugin, but the SQLite plugin, as per the error message.

Open an issue there.

**Comment by craig-at-rsg** [comment]
The sqlite plugin works fine for me - the problem is "OPEN database: _ionicstorage" ... "Could not open database"; the sqlite plugin works in general, but not for this plugin.

**Comment by mhartington** [comment]
`_ionicstorage` is just the generic name of the DB driver, nothing specific here. 

If you can make a demo that can replicate the issue, please post a repo here.

**Comment by craig-at-rsg** [comment]
I suspect adding _androidDatabaseProvider: 'system'_ to the params when creating the _ionicstorage database would fix the problem.

**Comment by mhartington** [comment]
>If you can make a demo that can replicate the issue, please post a repo here.

**Comment by jainank** [comment]
hey @mhartington 

I am also facing the same issue on Android 11 (Api 30) however it works fine for (API 29).
Here is my analysis:

this code throws errors
SQLiteNative.sqlc_db_open(this.dbfilename, this.openflags)
in file
https://github.com/liteglue/Android-sqlite-connector/blob/master/src/io/liteglue/SQLGDatabaseHandle.java
Line 14

**Comment by mhartington** [comment]
@jainank Open an issue on that sqlite plugin repo.

Ionic Storage is just a wrapper around local forage, it does not actually provide the native sqlite storage APIs.

**Comment by FireStormHR** [comment]
The package in question has been updated and the issue described has been resolved in v6.0.0
 https://github.com/storesafe/cordova-sqlite-storage/issues/954#issuecomment-804478983
